Cargo.toml

```
@@ -1,5 +1,9 @@
 [workspace]
-members = ["crates/ef-testing", "crates/sequencer"]
+members = [
+  "crates/ef-testing",
```

```suggestion
  "crates/*",
```
does this work in rust?

---

I think it does, didn't know, nice!